linera-execution/src/wasm/mod.rs

```diff
@@ -223,7 +223,7 @@ pub enum WasmExecutionError {
     CreateWasmtimeEngine(#[source] anyhow::Error),
     #[cfg(with_wasmer)]
     #[error(
-        "Failed to execute Wasm module in Wasmer. This may be caused by panics or insufficient fuel."
+        "Failed to execute Wasm module in Wasmer. This may be caused by panics or insufficient fuel. {0}"
```

Comment:
In my experience, the error was always the same here (hence the vague explanation). Perhaps something to test again after wasmer is updated?

Reply:
I think this helps at the GraphQL level - the inner error wasn't being rendered in the response otherwise.